### DISCUSSION TITLE
Right click issue

### DISCUSSION THREAD

**Discussion by cpelon** [nohal/dashboardsk_pi#60] [General]
Hi Pavel,
Thank you for your work on this project, I am very excited to have found it. I have compiled and running on my ubuntu box and running OPCN 5.9.0.  I have a hopefully quick question I can't seem to figure out.  When I enabled DSK I only have the right click menu for DSK, I don't have the OCPN right click menu. Am I missing something simple?

**Comment by nohal** [comment]
That is a bug in the current master code. Will get back to it next week.

**Comment by nohal** [comment]
Fixed.

**Comment by cpelon** [comment]
Yes, that fixed it. Thanks!
